app/src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+
+const createRow = number => {
+  const row = [];
+  let i = 0;
+  while (i < number) {
+    row.push(false);
+    i++;
+  }
+  return row;
+};
+
+function App() {
+  const [grid, setGrid] = useState(createRow(5000));
+  const [found, setFound] = useState(false)
+
+  const activeCeils = idx => {
+    const newGrid = [...grid];
+    newGrid[idx] = !grid[idx];
+    setGrid(newGrid);
+  };
+  
+  const heart = () => console.log("RUN!")
+
+  found && setInterval(() => {
```

Comment:
Not sure what the heart function is supposed to do in this case. Can you explain?